app/code/Magento/Backup/Controller/Adminhtml/Index/Create.php

```diff
@@ -55,7 +55,10 @@ public function execute()
             $this->_coreRegistry->register('backup_manager', $backupManager);
 
             if ($this->getRequest()->getParam('maintenance_mode')) {
-                if (!$this->maintenanceMode->set(true)) {
+                $this->maintenanceMode->set(true);
+                $isActive = $this->maintenanceMode->isOn();
```

Comment:
I think `$isActtive` is an unnecessary variable

Reply:
Hi @sivaschenko,

Did you mean I should code like this: 
 if(!$this->maintenanceMode->isOn()) ?

And Travis is passed in new solution where in previous case it was failed.

Please guide me for the right way. 

Thanks in advance.